packages/core/src/components/Tooltip/index.tsx

```diff
@@ -314,23 +314,17 @@ export class Tooltip extends React.Component<TooltipProps & WithStylesProps, Too
           marginTop: above ? -(tooltipHeight + targetRect.height + distance) : distance,
           textAlign: align,
         })}
+        onMouseEnter={handleMouseEnter}
```

Comment:
can you elaborate on why this change was needed? Seems fine, wanted to check that the `margin` areas didn't trigger the event listeners tho (like to the top/left of the actual content)

Reply:
>wanted to check that the margin areas didn't trigger the event listeners tho (like to the top/left of the actual content)
I can assure this is ok with a gif (hacked in a larger margin so we can see the gif easily). 
![margins-no-problem](https://user-images.githubusercontent.com/17525561/88339449-e7b67a80-ccee-11ea-8950-50563f6c408e.gif)

>can you elaborate on why this change was needed? 
Yesterday Erik and I ran into a weird case where if the mouse was on the trigger while the popup was opening in the direction the mouse came from, there wouldn't be a continuity between  mouse enter trigger -> mouse leave trigger (because of the popup animation) -> mouse entering popup -> mouse leaving popup (when animations passes the cursor) -> mouse back on trigger. 

Oddly enough, I don't seem to be encountering that today if I move the mouse handlers back down to the content div. But the first screenshot shows margins weren't causing an issue with the current implementation, so I think the handlers should be fine on either div.

![continuity](https://user-images.githubusercontent.com/17525561/88340174-33b5ef00-ccf0-11ea-8c25-9473ba9a0a49.gif)